architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,42 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with version 3.11, you can specify Security Context Constrains as a
+resource that is handled by your roles or cluster roles. This allows you to
+scope the access to your SCCs for users/groups locally to a certain project
+compared to previously assigning users/groups to an SCC, which had a
+cluser-wide impact.
+
+To include access to SCCs for your role, you need to specify the following lines
+and values:
+.Role-Based Access to SCCs
+[source,yaml]
+----
+rules:
+  apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+  resourceNames:
+  - myPermittingSCC
+----
+A role with such a rule would allow the subjects that are bound to it with a
+rolebinding or a clusterrolebinding to use the user-defined SCC called
+*myPermittingSCC*.
+
+[NOTE]
+====
+The RBAC system won't allow a user to create a rolebinding that would add more
+privileges than the user has themselves. Since the RBAC system works differently
```

Comment:
This phrase does not flow well.
But I am not entirely sure how to improve it bit by bit, So I'll propose a different take:
Because RBAC is deisgned to prevent escalation, even project administrators will be unable to grant access to SCC because they are not allowed, by default, to use the verb "use" on SCC resources, including the restricted SCC.